components/org.wso2.carbon.identity.oauth.endpoint/src/main/java/org/wso2/carbon/identity/oauth/endpoint/util/EndpointUtil.java

```diff
@@ -828,11 +832,36 @@ public static String getUserConsentURL(OAuth2Parameters params, String loggedInU
             }
         } catch (UnsupportedEncodingException e) {
             throw new OAuthSystemException("Error while encoding the url", e);
+        } catch (IdentityOAuth2Exception e) {
+            throw new OAuthSystemException("Error retrieve Service Provider for clientId:" + params.getClientId() , e);
         }
 
         return consentPageUrl;
     }
 
+    private static ServiceProvider getServiceProvider(OAuth2Parameters params) throws IdentityOAuth2Exception {
+        ServiceProvider sp = null;
```

Comment:
Follow consistent formatting.

Either leave a new line after each method declaration or don't leave at all. 
Follow the same convention everywhere to be consistent.

Reply:
resolved by https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/2030/commits/43c3a582c5d41fc44d849e3f3af1be6550d893ee